### DISCUSSION TITLE
Error: Can't open MySQL db [Solar] : Authentication plugin 'caching_sha2_password' cannot be loaded: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by scottevo** [SBFspot/SBFspot#615] [General]
I receive this error when trying to write to MySQL db

Error: Can't open MySQL db [Solar] : Authentication plugin 'caching_sha2_password' cannot be loaded: The specified module could not be found.

**Comment by sbf-** [comment]
This is not an SBFspot issue...
Possible fix can be found [here](https://stackoverflow.com/questions/49194719/authentication-plugin-caching-sha2-password-cannot-be-loaded)
